BuildResidentialHPXML/measure.rb

```diff
@@ -1382,6 +1382,72 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('W')
     args << arg
 
+    perf_data_capacity_type_choices = OpenStudio::StringVector.new
+    perf_data_capacity_type_choices << 'Capacity'
+    # perf_data_capacity_type_choices << 'CapacityFractionOfNominal'
+
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('perf_data_capacity_type', perf_data_capacity_type_choices, false)
+    arg.setDisplayName('Detailed Performance Data: Capacity Type')
+    arg.setDescription('Capacity type of detailed performance data.')
+    arg.setUnits('Capacity')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heating_perf_data_outdoor_temperatures', false)
+    arg.setDisplayName('Detailed Performance Data: Heating Outdoor Temperatures')
+    arg.setDescription('Outdoor temperatures of heating detailed performance data. If multiple performance data points, use a comma-separated list.')
```

Comment:
A few suggestions:
- For all these arguments, end the first sentence with "...... detailed performance data **if available**." to make it clearer that this is optional.
- For all these arguments, add a sentence "Applies only to variable-speed air-source HVAC systems (central air conditioners, mini-split air conditioners, air-to-air heat pumps, and mini-split heat pumps)."
- For `heating_perf_data_outdoor_temperatures`, add a sentence: "One of the outdoor temperatures must be 47 deg-F."
- For `cooling_perf_data_outdoor_temperatures`, add a sentence: "One of the outdoor temperatures must be 95 deg-F."